src/Task/StaticCacheFullBuildTask.php

```diff
@@ -3,30 +3,32 @@
 namespace SilverStripe\StaticPublishQueue\Task;
 
 use DateTime;
-use SilverStripe\Control\Director;
-use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\BuildTask;
-use SilverStripe\Dev\Deprecation;
+use SilverStripe\PolyExecution\PolyOutput;
 use SilverStripe\ORM\DataList;
 use SilverStripe\ORM\FieldType\DBDatetime;
 use SilverStripe\StaticPublishQueue\Job\StaticCacheFullBuildJob;
 use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
 use Symbiote\QueuedJobs\Services\QueuedJob;
 use Symbiote\QueuedJobs\Services\QueuedJobService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 
 class StaticCacheFullBuildTask extends BuildTask
 {
-    protected $title = 'Static Cache Full Build';
+    protected static string $commandName = 'static-cache-full-build';
```

Comment:
```suggestion
    protected static string $commandName = 'StaticCacheFullBuildTask';
```

Reply:
Discussion about this and other task command names is being held in the main issue.